package.json

```diff
@@ -42,7 +44,12 @@
     "dependencies": {
         "eslint-config-airbnb": "^18.2.0",
         "eslint-config-airbnb-base": "^14.2.0",
-        "eslint-config-prettier": "^6.12.0"
+        "eslint-config-prettier": "^6.12.0",
+        "lodash": "^4.17.20",
+        "request": "^2.88.2",
+        "sequelize": "^5.22.3",
+        "umzug": "^1.12.0",
+        "winston": "^3.3.3"
```

Comment:
This will affect `cloudify-ui-common` uses in frontend packages (bundle size should not be affected, but just by longer dependencies download time in `npm install`), but it'll be hard to avoid it, right?

Reply:
The only way I can think of would be to have a separate npm package